web/profiles/joinup/config/install/core.entity_form_display.rdf_entity.collection.propose.yml

```diff
@@ -22,11 +22,83 @@ dependencies:
     - image.style.thumbnail
     - rdf_entity.rdfentity.collection
   module:
+    - field_group
     - image
     - inline_entity_form
     - link
     - rdf_entity
     - text
+third_party_settings:
+  field_group:
+    group_general:
+      children:
+        - group_main
+        - group_description
```

Comment:
Yes, but we don't know yet from this ticket if we need to implement some JS for dynamic item summaries. See http://drupal.stackexchange.com/questions/68844/how-do-i-add-a-menu-subtitle-to-field-sets-displayed-as-vertical-tabs. Or it will be only a static description stored in the field group. Who's gonna tell us? I think we nee dto raise this question in a follow-up. It's not only about styling.